tests/modules/images/dominant-color-images/dominant-color-image-editor-gd-test.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Tests for dominant-color module.
+ * Tests for Dominant Color Images module.
  *
  * @package performance-lab
  * @group   dominant-color
```

Comment:
```suggestion
 * @group   dominant-color-images
```

Reply:
Addressed in https://github.com/WordPress/performance/pull/708/commits/68fa2df94d143ddbba86815232e32e502299bd9e